azerothcore/azerothcore-wotlk
# fix(DB/SAI): Add Gnomish Battle Chicken combat SAI

## Changes Proposed:
-  Battle Chicken casts Battle Squawk w/ 25% chance in combat on self
-  Battle Chicken casts Chicken Fury w/ 50% chance in combat on self

## Issues Addressed:
#9408

## Tests Performed:
Untested

## How to Test the Changes:
1. Use trinket Gnomish Battle Chicken
2. Attack mob with Chicken out
3. Wait for procs.
4. Invite some friends to see if they get the Haste buff.


## Known Issues and TODO List:

I pulled 25% and 50% out of my ass.  There is no concrete information on the proc rates - but the Haste proc has historically procced less than the Enrage proc.
